test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -454,13 +454,16 @@ void UcxContext::add_connection(UcxConnection *conn)
 {
     assert(_conns.find(conn->id()) == _conns.end());
     _conns[conn->id()] = conn;
+    UCX_LOG << "added " << conn->get_log_prefix() << " to connection map";
```

Comment:
maybe not add these logs? seems too much IMO..

Reply:
but it is printed twice per conenction:
- when a new connection created
- when an existing connection removed
IMO, it'd help me to find out the cause of the "timeout waiting for replies" problem more quickly w/o even checking source code and understanding the flow.
wdyt?

Reply:
maybe add specific message that some packet was dropped/ignored because not found in conn map?